ghost/core/core/server/api/endpoints/recommendations.js

```diff
@@ -0,0 +1,71 @@
+const recommendations = require('../../services/recommendations');
+
+module.exports = {
+    docName: 'recommendations',
+
+    browse: {
+        headers: {
+            cacheInvalidate: false
+        },
+        options: [],
+        permissions: true,
+        validation: {},
+        async query() {
+            return await recommendations.controller.listRecommendations();
+        }
+    },
+
+    add: {
+        statusCode: 201,
+        headers: {
+            cacheInvalidate: false
+        },
+        options: [],
+        validation: {},
```

Comment:
IMO we should just do the validation in the controller (which is already the case), and not in the endpoints. Otherwise we have two places where we check the body and have to do the validation. We also almost never implement this correctly and completely for other endpoints (e.g., in the offers endpoints: `ghost/core/core/server/api/endpoints/offers.js`)